benchmark/cluster_containers_setup.py

```diff
@@ -173,6 +208,8 @@ def __init__(self, reactor, env, options):
         self.cluster_cert = certificates_path.child(b"cluster.crt")
         self.user_cert = certificates_path.child(b"user.crt")
         self.user_key = certificates_path.child(b"user.key")
```

Comment:
This is not par of the refactoring, but it was a bug. We didn't get the initial number of containers and datasets, so we could exit the loop where we wait until all the containers and dataset were created before it really happened, if we already had some datasets and containers in the cluster already.